content/applications/finance/accounting/reporting/overview/customize.rst

```diff
@@ -1,90 +1,252 @@
+   creation interface.
 
-Activate the developer mode
-===========================
+Create a report
+===============
 
-In order to have access to the financial report creation interface, the
-:ref:`developer mode <developer-mode>` needs to be activated.
+In order to create a new report, go to :menuselection:`Accounting app --> Configuration -->
+Management: Accounting Reports`. From here, you can either create a **root report** or a
+**variant**.
 
-Create your financial report
-============================
+.. image:: customize/engine-accounting-reports.png
+   :align: center
+   :alt: Accounting reports engine.
+
+Root reports
+------------
+
+Root reports can be regarded as generic, neutral accounting reports. They serve as models on which
+local accounting versions are built. A tax report for Belgium and the US would both use the same
+generic version as a base and adapt it for their domestic regulations.
+
+When creating a new root report, you need to create a **menu item** for it. To do so, open the
+report and then on that same report, click on :menuselection:`Action --> Create Menu Item`. Refresh
+the page and the report is now available under :menuselection:`Accounting --> Reporting`. If a
+report has no root report, it is considered to be a root report itself.
 
-First, you need to create your financial report. To do that, go to
-:menuselection:`Accounting --> Configuration --> Financial Reports`
+.. note::
+   Cases that require creating a new root report are rare; such as when the tax authorities of a
+   country require a new and specific type of report.
 
-.. image:: customize/customize02.png
+.. image:: customize/engine-create-menu-item.png
    :align: center
+   :alt: Create Menu Item button.
+
+.. seealso::
+    List of built-in main :doc:`root reports <../overview/main_reports>`.
+
+Variants
+--------
 
-Once the name is entered, there are two other parameters that need to be
-configured:
+Variants are country-specific versions of root reports and, therefore, always refer to a root
+report. To create a variant, simply select a generic (root) report in the :guilabel:`Root Report`
+field when creating a new report.
 
--  **Show Credit and Debit Columns**
+When a root report is opened from one of the accounting app's main menus, all variants available for
+that root are displayed in the variant selector in the top right corner of the view. In the
+following image, :guilabel:`VAT Report (BE)` is the variant of the root :guilabel:`Generic Tax
+report`.
+
+.. image:: customize/engine-variant.png
+   :align: center
+   :alt: Report variant selection.
+
+Lines
+-----
+
+After having created a report (either root or variant), you need to fill it with lines. You can
+either create a new one by clicking on :guilabel:`Add a line`, or modify an existing line by
+clicking on it. All lines *require* a :guilabel:`Name`, and can have an optional additional
+:guilabel:`Code` (of your choice) if you wish to use their value in formulas.
```

Comment:
I understand that we want to see all the lines on the `account.report` form, but don't we want to see the direct children on the `account.report.line` form too (probably in a notebook)?
It would help when building the hierarchy of the report.

Reply:
For now, we don't do that, to avoid charging the UI more. 
I agree the creation of the hierarchy is a bit painful. Since it's a debugging view used by only few people, it's not a priority, but we're considering (POs and I) a js widget to ease this, and avoid having to set the parent manually when modifying the structure.